pkg/kubelet/kubelet.go

```diff
@@ -2513,6 +2542,19 @@ func (kl *Kubelet) syncLoopIteration(ctx context.Context, configCh <-chan kubety
 			}
 			klog.V(4).InfoS("SyncLoop (housekeeping) end", "duration", duration.Round(time.Millisecond))
 		}
+	case <-kl.nodeRegistrationCh:
```

Comment:
What ensures that the mirror / static pods are handled first? Since this is part of `syncLoopIteration` won't the pods also be sent over `configCh` as well and handled there as well?

Reply:
I thought podWorker would only handle one request at a time for a particular pod. So does it matter?